check50/__main__.py

```diff
@@ -238,15 +286,20 @@ def main():
                         metavar="FILE",
                         help=_("file to write output to"))
     parser.add_argument("-v", "--verbose",
+                        action="store_true",
+                        help=_("shows any print statements in checks if running locally, and shows which dependencies get installed"))
+    parser.add_argument("--log-level",
                         action="store",
-                        nargs="?",
-                        default="",
-                        const="info",
-                        choices=[attr for attr in dir(logging) if attr.isupper() and isinstance(getattr(logging, attr), int)],
-                        type=str.upper,
-                        help=_("sets the verbosity level."
-                               ' "INFO" displays the full tracebacks of errors and shows all commands run.'
-                               ' "DEBUG" adds the output of all command run.'))
+                        default="warning",
+                        choices=[level.name.lower() for level in LogLevel],
+                        type=str.lower,
+                        help=_("sets the log level."
+                               ' "warning" displays usage warnings'
+                               ' "info" shows all commands run.'
+                               ' "debug" adds the output of all command run.'))
+    parser.add_argument("--ansi-log",
+                        action="store_true",
+                        help=_("display log in ansi output mode"))
```

Comment:
Why is that separate from `--log-level` again?

Reply:
`--ansi-log` shows the student facing logs in the ansi output of the checks:

```
running clang...
looking for output...
```

`--log-level` controls "developer" (check writers, extension writers, Kareem's of this world) logs.